staging/src/k8s.io/apimachinery/pkg/api/meta/help.go

```diff
@@ -221,6 +245,9 @@ func extractList(obj runtime.Object, allocNew bool) ([]runtime.Object, error) {
 	if err != nil {
 		return nil, err
 	}
+	if items.IsNil() {
```

Comment:
does this change any existing behavior that expects a zero-length list to be returned?

Reply:
Can't be 100 percent sure. However I expect append and growslice to still work on nil.